INSTALL.md

```diff
@@ -583,6 +583,10 @@ access to algorithm internals that are not normally accessible.
 Additional information related to ACVP can be found at
 <https://github.com/usnistgov/ACVP>.
 
+### no-apps
+
+Do not build apps, e.g. the openssl program. This is handy for minimization.
```

Comment:
Probably we should make it clear that this option also disables the tests - so "make test" will no longer work.

Reply:
Ok, note added.